python/whylogs/experimental/core/udf_schema.py

```diff
@@ -202,6 +204,30 @@ def _reset_udfs(reset_metric_udfs: bool = True) -> None:
     _resolver_specs = defaultdict(list)
 
 
+def register_validator_udf(
+    col_names: List[str],
+    condition_name: Optional[str] = None,
+    actions: Optional[List[Callable[[str, str, Any, Any], None]]] = None,
+    namespace: Optional[str] = None,
+    schema_name: str = "",
+) -> Callable[[Any], Any]:
+    def decorator_register(func):
+        global _validator_udfs
+        name = condition_name or func.__name__
+        name = f"{namespace}.{name}" if namespace else name
+        for col in col_names:
+            validator = ConditionValidator(
+                name=name,
+                conditions={func.__name__: func},
```

Comment:
This is actually very confusing, because we have:
- condition name
- condition validator name

Which are not necessarily the same.  One condition validator can have multiple conditions. By using the register decorator, it happens to be 1 condition to 1 validator, so maybe in this case we could have the same name for both?